test/end-to-end/core.sh

```diff
@@ -426,6 +426,24 @@ os::cmd::expect_success "oc patch dc/failing-dc-mid -p '{\"status\":{\"latestVer
 os::cmd::expect_success_and_text 'oc logs --version=1 dc/failing-dc-mid' 'test mid hook executed'
 os::cmd::expect_success_and_text 'oc logs --previous dc/failing-dc-mid'  'test mid hook executed'
 
+os::log::info "Rollout latest"
```

Comment:
@mfojtik I though we are trying to move away from bash end-to-end in favor of extended tests written in go or did I got it wrong?

Reply:
@tnozicka this is really a CLI test, i'm pretty sure we have extended tests for generic trigger and i don't want to bloat our already bloated deployments extended test with this. The rollout latest is just CLI.

Reply:
CLI is available in our extented suite

Reply:
@tnozicka but those tests take 2-4x more time to run :-)

Reply:
Anything that is CLI / api / controller can safely be in this suite.

Reply:
Most CLI things should be tested here unless they need to deal with complicated life.